tests/test_dataset.py

```diff
@@ -247,16 +208,75 @@ def test_iso_language_mappings(languages, iso_languages_expected):
 
 
 @pytest.mark.parametrize(
-    'dbs',
+    "dbs",
     [
-        ['minimal_db', 'medium_db'],
+        ["minimal_db", "medium_db"],
     ],
 )
 def test_iso_language_property(dbs, cache, request):
     """Test ISO 639-3 language mapping property."""
     dbs = [request.getfixturevalue(db) for db in dbs]
+
     datasets = [
-        audbcards.Dataset(db.name, pytest.VERSION, cache)
+        audbcards.Dataset(db.name, pytest.VERSION, cache_root=cache)
         for db in dbs
     ]
     _ = [dataset.iso_languages for dataset in datasets]
+
+
+@pytest.fixture
+def constructor(tmpdir, medium_db, request):
+    """Fixture to test memory/timing footprint of constructor."""
+    db = medium_db
+    dataset_cache = audeer.mkdir(tmpdir, "cache")
+    dataset_cache_filename = audbcards.Dataset._dataset_cache_path(
+        db.name, pytest.VERSION, dataset_cache
+    )
+
+    ex0 = os.path.exists(dataset_cache_filename)
+
+    ds_uncached = audbcards.Dataset(
+        db.name, pytest.VERSION, cache_root=dataset_cache
+    )
+
+    ex1 = os.path.exists(dataset_cache_filename)
+
+    ds_cached = audbcards.Dataset(
+        db.name, pytest.VERSION, cache_root=dataset_cache
+    )
+
+    ex2 = os.path.exists(dataset_cache_filename)
+
+    constructor = (ds_uncached, ds_cached, [ex0, ex1, ex2])
+
+    return constructor
+
+
+@pytest.mark.usefixtures("constructor")
+class TestConstructor(object):
+    """Test constructor class method.
+
+    Testing of
+
+    - creation footprints of Dataset instances
```

Comment:
We are no longer testing this here, correct?

So maybe change to
```suggestion
    - existence of cache files
```